packages/payments-plugin/e2e/mollie-payment.e2e-spec.ts

```diff
@@ -603,228 +634,38 @@ describe('Mollie payments with useDynamicRedirectUrl=false', () => {
             expect(createShipmentBody).toBeDefined();
             expect(order.state).toBe('PaymentSettled');
         });
-    });
-
-    it('Should add an unusable Mollie paymentMethod (missing redirectUrl)', async () => {
```

Comment:
All tests below are removed, because they check the correct usage of `useDynamicRedirectUrl`, but that option has been removed.